loleaflet/src/control/Control.Header.js

```diff
@@ -597,13 +597,13 @@ L.Control.Header = L.Control.extend({
 			canvas.width = Math.floor(value * scale);
 			if (!isCorner)
 				this._canvasWidth = value;
-//			console.log('Header._setCanvasSizeImpl: _canvasWidth' + this._canvasWidth);
+			//			console.log('Header._setCanvasSizeImpl: _canvasWidth' + this._canvasWidth);
```

Comment:
This doesn't seem right. It will look pretty weird when the `//` part is removed while debugging.